frontend/src/components/form/index.tsx

```diff
@@ -77,11 +82,11 @@ Form.Spinner = function FormSpinner({ ...restProps }: any) {
   return <Spinner {...restProps} />
 }
 
-Form.Submit = function FormSubmit({ loading }: any) {
+Form.Submit = function FormSubmit({ loading, children }: any) {
   return (
     <Submit>
       {!loading ? (
```

Comment:
Seems like it would be more readable to reverse the logic. 

`isLoading ? <Spinner /> : children`